docs/source/reference/kubernetes/kubernetes-ports.rst

```diff
@@ -22,17 +22,17 @@ If your cluster does not support LoadBalancer services, SkyPilot can also use `a
 
 .. _kubernetes-loadbalancer:
 
-LoadBalancer Service
+LoadBalancer service
```

Comment:
I guess in the Kubernetes context, `Service` should be capitalized. You can see in the Kubernetes docs linked below